core/src/stores/sqlite.rs

```diff
@@ -1005,7 +1005,13 @@ impl Store for SQLiteStore {
         project: &Project,
         data_source_id: &str,
         document_id: &str,
+        // NOTE: doesn't work on SQLITE
+        get_previous_version: bool,
     ) -> Result<Option<Document>> {
+        if get_previous_version {
+            return Err(anyhow!("get_previous_version not supported on SQLITE"));
```

Comment:
Wondering if that's fine as we're deprecating ?

Reply:
YES